src/parcsr_mv/HYPRE_parcsr_matrix.c

```diff
@@ -408,3 +408,18 @@ HYPRE_ParCSRMatrixMatvecT( HYPRE_Complex      alpha,
                alpha, (hypre_ParCSRMatrix *) A,
                (hypre_ParVector *) x, beta, (hypre_ParVector *) y) );
 }
+
+/*--------------------------------------------------------------------------
+ * HYPRE_ParCSRMatrixDiagScale
+ *--------------------------------------------------------------------------*/
+
+HYPRE_Int
+HYPRE_ParCSRMatrixDiagScale(HYPRE_ParCSRMatrix A,
```

Comment:
This is an example of an external matrix interface that we should discuss as a team before making it available.

Reply:
Yes, same as `axpyz`. I'll remove this public API from this PR

Reply:
I'm using it in the IJ driver, should I replace it with `hypre_ParCSRMatrixDiagScale`?

Reply:
Yes.  We try to use the drivers as a means to test user interfaces, but this is an exception (for now, since we may end up adding the external interface later).

Reply:
Ok! Thanks!!